packages/cardano-services/src/cli.ts

```diff
@@ -107,17 +107,19 @@ const runServer = async (message: string, loadServer: () => Promise<HttpServer>)
 };
 
 withCommonOptions(
-  withOgmiosOptions(
-    withPostgresOptions(
-      program
-        .command('start-projector')
-        .description('Start a projector')
-        .argument(
-          '[projectionNames...]',
-          `List of projections to start: ${Object.values(ProjectionName).toString()}`,
-          projectionNameParser
-        ),
-      ''
+  withHandlePolicyIdsOptions(
```

Comment:
Minor minor minor minor: adding `withHandlePolicyIdsOptions` as inner _option modifier function_ would have a lesser impact on the diff making it a bit more clear.

Reply:
af17ac42b57672c085e8d89d7f2565a979aaad0c should fix this